cmd/inspektor-gadget/cmd/health.go

```diff
@@ -54,13 +61,13 @@ func runHealth(cmd *cobra.Command, args []string) {
 	fmt.Fprintln(w, "NODE\tSTATUS\t")
 
 	for _, node := range nodes.Items {
-		line := fmt.Sprintf("%s\t%s\t", node.Name, execPodQuick(client, node.Name, "/bin/gadget-node-health-check.sh"))
+		line := fmt.Sprintf("%s\t%s\t", node.Name, execPod(client, node.Name, "/bin/gadget-node-health-check.sh", true))
 		fmt.Fprintln(w, line)
 	}
 	w.Flush()
 }
 
-func execPodQuick(client *kubernetes.Clientset, node string, podCmd string) string {
+func execPod(client *kubernetes.Clientset, node string, podCmd string, capture bool) string {
```

Comment:
could you move all of the helper functions that are not related to the `inspektor-gadget health` subcommand to another file than `health.go`? It's my fault, I put the helper functions there in the first place, but maybe that's a good opportunity to clean up this.

Reply:
done